src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -94,8 +96,24 @@ public Film getFilmById(@PathVariable Long id) {
      * @param count количество попавших в топ фильмов(Если не указано, то 10)
      * @return возвращает список фильмов с количеством лайков (От большего к меньшему)
      */
+
```

Comment:
Обрати внимание на документацию, она пишется иначе:
```suggestion
/**
 * Здесь описание метода
 *
 * @param переменная, которая передаётся в параметре метода
 * @return переменная, которая возвращается(если она есть)
 * @throws если выбрасывается исключение
 */
```

Reply:
я не писал документацию, это все было  до меня. я не в курсе как ее писать

Reply:
> Обрати внимание на документацию, она пишется иначе:
я не писал документацию, это все было до меня. я не в курсе как ее писать

Reply:
а увидел, оставил коммент, это я накидал чтоб в табличку вставить что сделано, забыл убрать